packages/flutter_tools/test/general.shard/drive/drive_service_test.dart

```diff
@@ -366,6 +367,25 @@ void main() {
     );
     await driverService.stop();
   });
+
+  testWithoutContext('WebDriver error message includes link to documentation', () async {
+    const String link = 'https://flutter.dev/docs/testing/integration-tests#running-in-a-browser';
+    final DriverService driverService = WebDriverService(
+      dartSdkPath: 'dart',
+      processUtils: ProcessUtils(
+        processManager: FakeProcessManager.empty(),
+        logger: BufferLogger.test(),
+      ),
+    );
+
+    expect(() => driverService.startTest(
+      'foo.test',
+      <String>[],
+      <String, String>{},
+      PackageConfig(<Package>[Package('test', Uri.base)]),
+      browserName: 'chrome',
+    ), throwsToolExit(message: RegExp(' $link ')));
```

Comment:
I would probably make this test more strict. It's not just that there's a link that matters, it's also that the text is correct around the link, right?

Reply:
Good point, I made the test more strict by including the preamble 👍